app/views/posts/show.html.erb

```diff
@@ -22,4 +23,15 @@
       </div>
     <% end %>
   </div>
+  <div class="text-center p-4">
+  <h3>add like</h3>
+  
+     <h3>Add a comment: </h3>
+       <%= form_for [@post, Comment.new], url: "/comments/#{@post.id}" do |form| %>
+        <div>
+          <%= form.text_area :text, placeholder: "Add comment", class: "form-control"%>
+        </div>
+        <%= form.submit "Add Comment", class: "btn btn-outline-secondary rounded-pill m-2" %>
+      <% end %>
```

Comment:
- [ ] After creating a post through the terminal, I was unable to create a comment. I have added a video below :+1: So kindly ensure that users are able to create comments successfully. 
- [ ] Also, as I mentioned in my earlier comment the same applies to comments. I noticed that both routes for `comments#create` and  `comments#new` have been included in the `routes.rb` file. However, there is no view template for `comments#new`.
- [ ] So kindly ensure that there is a separate form in the `views/comments/new.html.erb` file that accepts input for creating comments. :+1: 
- Missing Template error
![Screenshot from 2023-10-07 17-12-42](https://github.com/over-geek/blog-app/assets/102699958/b58039e7-07a8-483e-9feb-5a26062de799)
- Comment Creation Video

[Screencast from 10-07-2023 05:21:07 PM.webm](https://github.com/over-geek/blog-app/assets/102699958/27b0753d-7fd0-4efa-896d-0a2669df20c4)